lib/tasks/stop_import.rake

```diff
@@ -10,11 +10,33 @@ STRING_COLUMN_ROW_NAMES = {
   mounting:           %w(stp_ud_mounting_pvta_pole         stp_ud_mounting_pole_other        stp_ud_mounting_structure  ),
   schedule_holder:    %w(stp_ud_schedule_holder_on_pole    stp_ud_schedule_holder_in_shelter                            ),
   shelter:            %w(stp_ud_shelter_pvta               stp_ud_shelter_other              stp_ud_shelter_building    ),
-  sidewalk:           %w(stp_ud_sidewalk_more_than_36in    stp_ud_sidewalk_less_than_36in    stp_ud_sidewalk_no_sidewalk),
-  sign:               %w(stp_ud_sign_flag_stop             stp_ud_sign_missing               stp_ud_sign_needs_attention),
+  sidewalk_width:           %w(stp_ud_sidewalk_more_than_36in    stp_ud_sidewalk_less_than_36in    stp_ud_sidewalk_no_sidewalk),
+  # sign:               %w(stp_ud_sign_flag_stop             stp_ud_sign_missing               stp_ud_sign_needs_attention),
+  # apparently sign's not needed :/
   trash:              %w(stp_ud_trash_pvta                 stp_ud_trash_municipal            stp_ud_trash_other         )
 }
 
+STRING_COLUMN_OPTIONS = {
+  accessible: ['When necessary', 'Not recommended'],
+  bench: %w[PVTA Other],
+  curb_cut: ["Within 20'", 'No curb cut', 'No curb'],
+  lighting: ["Within 20'", "20' - 50'", 'None'],
+  mounting: ['PVTA pole', 'Other pole', 'City Pole', 'Utility Pole', 'Structure'],
+  mounting_direction: ['Towards street', 'Away from street'],
+  schedule_holder: ['On pole', 'In shelter'],
+  shelter: %w[PVTA Other Building],
+  shelter_condition: %w[Great Good Fair Poor],
+  shelter_pad_condition: %w[Great Good Fair Poor],
+  shelter_pad_material: %w[Asphalt Concrete Other],
+  shelter_type: %w[Modern Modern\ half Victorian Dome Wooden Extra\ large Other],
+  sidewalk_width: ['More than 36"', 'Less than 36"', 'None'],
+  # sign: ['Flag stop', 'Missing sign', 'Needs attention'],
+  # sign_type: ['Axehead (2014+)', 'Rectangle (<2014)', 'MGM + Axhead (2018+)'],
+  # sign type is not in hastus yet
```

Comment:
But it doesn't matter. If it's in the field guide, we need to be collecting the data. We have quite a while to make HASTUS match the field guide. This app needs to match it now.